components/common/Card.tsx

```diff
@@ -1,19 +1,20 @@
-import React from 'react'
+import React, { ComponentProps } from 'react'
 
 export type Variation = 'default' | 'error'
 
 export const Card = ({
-  children,
+  className = '',
   variation = 'default',
-}: {
-  children: React.ReactNode
+  ...props
+}: ComponentProps<'div'> & {
   variation?: Variation
 }) => {
-  let className = 'shadow rounded-sm p-2'
+  className += ' shadow rounded-sm p-2'
```

Comment:
don't think this needs to be changed but FYI we also have a handy `classNames` util in `@/lib/utils` if you wanna avoid string concatenation like that.